x10rf.cpp

```diff
@@ -147,15 +147,13 @@ void x10rf::RFXsensor(uint8_t rfxs_address,uint8_t rfxs_type, char rfxs_packet_t
 			default:
 				x10buff[3] = 0x00;
 			}
-	x10buff[3] << 4;
```

Comment:
Hi @arcadien, 
Why removing this bit shifting instruction ? 
This will change the result of this method and work anymore as expected

Reply:
afaik, this instruction has no side effect.
```
x10buff[0] = x10buff[0] << 4; // House code goes into the upper nibble
``` 
would have
would have.

Reply:
Indeed !